modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/api.ts

```diff
@@ -72,9 +74,17 @@ export async function getFields(fdsView: FDSViewType) {
 			return;
 		}
 
+		if (propertyKey.includes(localizablePropertySuffix)) {
+			return;
+		}
```

Comment:
This is fine :+1: 

Interestingly, Objects team chose to implement localization by duplicating schema properties. Extra table in DB makes perfect sense, but I don't think it was necessary to do this. It could easily be a dedicated property, or a naming convention. That's what I though it would be, just a `_i18n` in name. Oh well, be can adapt to it.